tests/unit/schema/wrappers/test_method.py

```diff
@@ -999,3 +1055,65 @@ def test_mixin_rule():
         "city": {},
     }
     assert e == m.sample_request
+
+
+@pytest.mark.parametrize(
+    "field_type, _type, expected",
```

Comment:
Use `pb_type` instead of `_type` since this is referring to the protobuf type and this avoids confusion with the [built-in function type](https://docs.python.org/3/library/functions.html#built-in-functions)